doc/reference/kernel/other/float.rst

```diff
@@ -113,6 +118,17 @@ is currently enabled in Zephyr applications on ARM Cortex-M
 architecture, minimizing interrupt latency, when the floating
 point context is active.
 
+When the MPU-based stack protection menchanism is not enabled, lazy stacking
+is always active in the Zephyr application. When the MPU-based stack protection
+is enabled, the followimg rules apply with respect to lazy stacking:
+
+* Lazy stacking is activated by default on threads that are pretagged with
+  :c:macro:`K_FP_REGS`
+* Lazy stacking is activated dynamically on threads that are pretagged with
+  :c:macro:`K_FP_REGS` as soon as the kernel detects that they are using the
+  floating point registers.
```

Comment:
Both cases describe "... on threads that are pretagged with". I guess that those should be two different cases?

Reply:
thanks, the second is _not_ pretagged, of course